news/764-bugfix.md

```diff
@@ -0,0 +1 @@
+Clean up the Python code lint after running pylint3
```

Comment:
Can you delete this file from your branch please -- this no longer exists on master since it's been moved into the main CHANGELOG during the recent release.